staging/src/k8s.io/cli-runtime/pkg/resource/kustomizevisitor.go

```diff
@@ -38,7 +38,7 @@ type KustomizeVisitor struct {
 // Visit passes the result of a kustomize build to a StreamVisitor.
 func (v *KustomizeVisitor) Visit(fn VisitorFunc) error {
 	kOpts := krusty.MakeDefaultOptions()
-	kOpts.DoLegacyResourceSort = true
+	kOpts.Reorder = krusty.ReorderOptionLegacy
```

Comment:
The legacy sorting is the default used by krusty, maybe it's better to leave it out.

Reply:
Yup, that's true. Good point.

Reply:
Actually double checked it and the default value is `ReorderOptionNone`: https://github.com/kubernetes-sigs/kustomize/blob/api/v0.13.1/api/krusty/options.go#L53
I would then say to keep it as it is proposed here.

Reply:
Hmm really strange, as the docs state that legacy is the default https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/sortoptions/

Reply:
Yeah, a couple more things are not aligned here. The code states that there are just 3 supported methods: `legacy`, `none` and `unspecified`. I guess that `fifo` could be an equivalent to `none`, but who knows, I am not familiar with this. What do you think the best solution is then? I would be up for the one that is proposed now and maybe raise an issue for docs team or something? Wdyt?